modules/rostests/readme.txt

```diff
@@ -6,19 +6,14 @@ This directory contains various tests for ReactOS.
 Make sure you also have a copy of the rest of the ReactOS
 source before you attempt to build anything in this module.
 It is to be placed under "modules" subdirectory of a trunk checkout.
-No additional changes to build files are necessary, it'll be picked up
-automatically.
+Module require to be enabled during "configure" process.
 
-To include "rostests" in your build folder:
-1. Copy the rostests folder into the reactos\modules folder, or
-2. Link reactos/modules/rostests to rostests
+To include module in your build folder, run configure script with flags -DENABLE_ROSTESTS=1
```

Comment:
To include **the** module..., run **the** configure script with **the** flags ...